### PR TITLE
perf: findLatestTransaction

### PR DESCRIPTION
**Pull request by kalgoop**
## Proposed changes

instead of sorting and then picking the latest transaction, loop over all transactions to find the latest transaction.
Reduces complexity from O(nlogn) to O(n)

## Types of changes

- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

File: src/renderer/services/synchronizer/wallets.js [hunk 1]
```diff
@@ -172,7 +172,13 @@ class Action {
   }
 
   findLatestTransaction (transactions) {
-    return last(sortBy(transactions, 'timestamp'))
+    var latestTransaction = transactions[0]
+
+    for (const i in transactions) {
+      if (transactions[i].timestamp > latestTransaction.timestamp) latestTransaction = transactions[i]
```
Comment (alexbarnsley) [inline src/renderer/services/synchronizer/wallets.js]: can you use curly brackets for if statements please, as it makes it more consistent across the codebase. ones without brackets are harder to read imo

**Comment by kalgoop** [comment]
@alexbarnsley fixed

**Comment by faustbrian** [comment]
https://github.com/ArkEcosystem/desktop-wallet/pull/878

**Comment by kalgoop** [comment]
@faustbrian this PR introduces changes different than those in #878 .
The former code and code after #878 both have complexity O(nlogn) for finding latest transaction. This PR achieved that in O(n)

**Pull request merged**